cfme/tests/configure/test_access_control.py

```diff
@@ -162,6 +163,16 @@ def test_user_remove_tag():
     user.delete()
 
 
+def test_delete_default_user():
```

Comment:
We just started documenting our tests and since this is a new test, it might be good to comment it like this:
```python
def test_foo(bar, baz):
    """This test foos bar with baz.

    This is lorem ipsum dolor sit amet etcetera,

    Prerequisities:
        * Acme
        * Emca

    Steps:
        * Foo bar with baz
        * Use acme
    """
    pass  # Here comes the test
```